.github/workflows/percy-issue-comment.yml

```diff
@@ -69,7 +69,7 @@ jobs:
 
       - name: Prepare cypress environment
         uses: ./.github/actions/prepare-cypress
-      - run: ./bin/build
+      - run: ./bin/build.sh
```

Comment:
e.g. does percy issue comment need to be checking our library licenses, building every single driver, etc? Probably not.